src/tables.jl

```diff
@@ -63,7 +62,8 @@ function schema_names(layer::AbstractFeatureLayer)
     featuredefn = layerdefn(layer)
     fielddefns = (getfielddefn(featuredefn, i) for i in 0:nfield(layer)-1)
     field_names = (Symbol(getname(fielddefn)) for fielddefn in fielddefns)
-    geom_names = (Symbol(getname(getgeomdefn(featuredefn, i-1))) for i in 1:ngeom(layer))
+    geom_names = collect(Symbol(getname(getgeomdefn(featuredefn, i-1))) for i in 1:ngeom(layer))
+    replace!(geom_names, Symbol("")=>Symbol("geom"), count=1)
```

Comment:
This could theoretically clash with an existing identifier, but I think it's very unlikely.

Reply:
Still would probably be good to check. So most files have a single geometry that has no name?

I'm thinking a bit about `geom` versus `geometry`. The latter is a bit longer, but may be more clear, and seems to be more in line with sf and geopandas.

Reply:
Yeah, that happens a lot. And that's inconvenient for accessing, as your normal access pattern `a.name` won't work. I'm fine with `geometry`.

Reply:
I'm good with `geometry` too

Reply:
> So most files have a single geometry that has no name?

Yeah, that's the historical mental model people had of features and geometries. It seems like there's some effort to move away from that towards properly supporting features with multiple geometries (across GDAL and geopandas), but it seems that those are more experimental, and I haven't seen any (i) good documentation on a coherent model of how it should work and (ii) evidence that rigorous tests have been introduced to ensure that it'll obey those contracts.

Reply:
It seems like we're getting more packages with geometry columns, and I think it would be good to all give them the same name. I think preference would be `geometry`. We now also need to pick something here: https://github.com/JuliaGeo/Shapefile.jl/pull/49. Only I see https://github.com/evetion/GeoDataFrames.jl uses `geom`. @evetion do you think it would be good to change it there as well?

And much less important, but for visual consistency, do we put the geometry column at the beginning or at the end? I see GeoDataFrames puts it at the end. Since it is generally not very interesting to look at text representations of geometries, you don't want it to take up to much space. Luckily #159 also helps with that. One reason to perhaps bring it to the front, is because at the end it may fall off, and it is an important column, just look at the beautiful DataFrame I just made in my narrow shell:

```
julia> DataFrame(sadfasdfsd=1:3, asdfasdgaererer="aahweifhwe", asdfsfe=3:5, as
dfsf3e=3:5, asdf2sfe=3:5, asdfs3634fe=3:5, asdfsf63e=3:5, asdfsf123e=3:5, asdf
sf21e=3:5, lastcol=3:5)
3×10 DataFrame
 Row │ sadfasdfsd  asdfasdgaererer  asdfsfe  asdfsf3e  asdf2sfe  asdfs3634fe ⋯
     │ Int64       String           Int64    Int64     Int64     Int64       ⋯
─────┼────────────────────────────────────────────────────────────────────────
   1 │          1  aahweifhwe             3         3         3            3 ⋯
   2 │          2  aahweifhwe             4         4         4            4
   3 │          3  aahweifhwe             5         5         5            5
                                                             4 columns omitted
```

Reply:
+1 for putting the geometry in the first few columns for the reasons @visr mentioned

Reply:
I'm happy to change `geom` to `geometry`. I have no preference for the order of columns.